packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.service.js

```diff
@@ -34,9 +35,11 @@ export default class FlavorsList {
       .then(({ flavors, prices }) => map(
         groupBy(flavors.filter(({ planCodes }) => !isNil(planCodes.hourly)), 'name'),
         groupedFlavors => new Flavor({
-          ...omit(groupedFlavors[0], ['id', 'region']),
+          ...omit(groupedFlavors[0], ['available', 'id', 'region']),
+          available: reduce(groupedFlavors,
```

Comment:
maybe use https://lodash.com/docs/4.17.15#some ?

`available: some(groupedFlavors, 'available')`

Reply:
Thanks a lot @frenautvh . This change is done :)